tests/index.html

```diff
@@ -36,11 +36,24 @@
   <script src="/shopify-buy.amd.js"></script>
   <script src="/tests.js"></script>
   <script>
-    Object.keys(require.entries).filter(function(name) {
-      return !!name.match(/-test$/);
-    }).forEach(function(name) {
-      require(name);
-    });
+    // The following will run actual tests.
+    // What we will do is get all modules defined. Pick up only modules that define tests
+    // based on the name of the module (module name ends in "-test")
+    
+    // It would be nice to just do `var moduleName = Object.keys(require.entries)`
+    // however IE<9 does not support Object keys so we'll need to use for(var i in require.entries)
+    // instead.
+    var modules = require.entries;
+
+    for (var moduleName in modules) {
+      // this moduleName is not defined in a proto
+      if (modules.hasOwnProperty(moduleName)) {
+        // is this module a test if so we'll require it
+        if(!!moduleName.match(/-test$/)) {
```

Comment:
There's no reason to booleanize here. The only time I'd do that is if I'm in a predicate function and I don't want to return an object. And even in that case: https://github.com/Shopify/javascript/blob/2ca4167bc12857787fad1d040ba203519d48d285/packages/shopify-codemod/docs/implicit-coercion-to-explicit.md